tools/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -233,6 +233,33 @@ class QualificationSuite extends FunSuite with BeforeAndAfterEach with Logging {
     }
   }
 
+  test("test skip gpu event logs") {
+    val qualLogDir = ToolTestUtils.getTestResourcePath("spark-events-qualification")
+    val logFiles = Array(s"$qualLogDir/dataset_eventlog")
```

Comment:
If dataset_eventlog is a GPU event log we need to generate a new one taht is CPU based because there should be other tests in there to deal with datasets.  Please check all of htem. I would prefer to call on gpu_eventlog or something to make it obvious what it is.

Reply:
Created CPU based dataset_eventlog and updated the relevant tests.  PTAL